src/components/panel/panel.js

```diff
@@ -923,9 +1043,23 @@ MdPanelRef.prototype.open = function() {
   return this._$q(function(resolve, reject) {
     var done = self._done(resolve, self);
     var show = self._simpleBind(self.show, self);
+    var setOpened = function(response) {
```

Comment:
@bradrich Promises continue if they resolve, which it does because this is synchronous and wrapped in a then. You never call reject here.